src/CMakeLists.txt

```diff
@@ -754,6 +754,10 @@ function(upm_module_init)
     endif (BUILDSWIGNODE)
     if (BUILDSWIGJAVA)
       upm_swig_java()
+
+      if (BUILDCORDOVABINDING AND NOT ";${JAVASWIG_BLACKLIST};" MATCHES ";${libname};")
```

Comment:
You could add a little comment explaining why this check is required!

Reply:
It's a complete copy of line 425. I will copy the comment before that line to here.